docs/_static/css/pyodide.css

```diff
@@ -2,3 +2,7 @@
     margin-right: 5px;
     opacity: 0.7;
 }
+
+code {
+    font-size: 100% !important;
```

Comment:
Which code blocks does this refer to. Might be preferable to use a more specific selector rather than the `!important` tag.

Reply:
It's overriding another selector that says
```css
code {
    font-size: 87.5% !important;
}
```
so I think there is no other choice here.